.tool-versions

```diff
@@ -1,2 +1,3 @@
 golang 1.20
-nodejs lts-hydrogen
+nodejs 18.18.2
+
```

Comment:
it's a bad practice and supported in new asdf versions to refer to an alias in .tool-versions, you must use the full version: https://github.com/asdf-vm/asdf-nodejs
```
 In asdf, we prioritize reproducibility, so you cannot use partial versions or codenames in a .tool-versions file.
```